pkg/httpmock/stub.go

```diff
@@ -18,6 +18,16 @@ type Stub struct {
 	matched   bool
 	Matcher   Matcher
 	Responder Responder
+	label     string
+}
+
+func (s *Stub) SetLabel(l string) *Stub {
```

Comment:
Obviously this could also just be a public field called Label. I figured method chaining looks a little nicer than assigning the result of `registry.Register` and then setting `x.Label` on a separate line.